dtfs-central-api/src/v1/controllers/utilisation-report-service/fee-record-correction/get-fee-record-correction.controller/index.ts

```diff
@@ -0,0 +1,60 @@
+import { ApiError, CurrencyAndAmount, RecordCorrectionReason } from '@ukef/dtfs2-common';
+import { HttpStatusCode } from 'axios';
+import { Response } from 'express';
+import { CustomExpressRequest } from '../../../../../types/custom-express-request';
+import { NotFoundError } from '../../../../../errors';
+import { FeeRecordCorrectionRepo } from '../../../../../repositories/fee-record-correction-repo';
+import { mapFeeRecordCorrectionEntityToResponse } from './helpers';
+
+/**
+ * Request type for the GET fee record correction endpoint.
+ */
+export type GetFeeRecordCorrectionRequest = CustomExpressRequest<{
+  params: {
+    correctionId: string;
+  };
+}>;
+
+/**
+ * Response body type for the GET fee record correction endpoint.
+ */
+export type GetFeeRecordCorrectionResponseBody = {
+  id: number;
+  bankId: string;
+  facilityId: string;
+  exporter: string;
+  reportedFees: CurrencyAndAmount;
+  reasons: RecordCorrectionReason[];
+  additionalInfo: string;
+};
+
+export type GetFeeRecordCorrectionResponse = Response<GetFeeRecordCorrectionResponseBody | string>;
+
+/**
+ * Controller for the GET fee record correction endpoint
+ * @param req - The request object
+ * @param res - The response object
+ * @returns The response object with a response status code and body
+ */
+export const getFeeRecordCorrection = async (req: GetFeeRecordCorrectionRequest, res: GetFeeRecordCorrectionResponse) => {
+  const { correctionId } = req.params;
+
+  try {
+    const feeRecordCorrectionEntity = await FeeRecordCorrectionRepo.findOneByIdWithFeeRecordAndReport(Number(correctionId));
+
+    if (!feeRecordCorrectionEntity) {
+      throw new NotFoundError(`Failed to find a fee record correction with id '${correctionId}'`);
```

Comment:
can we console error here too please

Reply:
I'm not sure this is necessary - this exception will be caught by the `catch` block just below which will log the message passed to the `NotFoundError` constructor

![Logs](https://github.com/user-attachments/assets/bdd6ac3a-ef74-4945-8360-8f7fa6d912d6)